css/src/style.scss

```diff
@@ -20,101 +20,75 @@ $white: #fff;
 	*/
 
 #app-navigation {
-	li {
-		transition: max-height 300ms ease 0s;
-		max-height: 100px;
-
-		> a {
-			padding-right: 20px;
-		}
+	> ul {
+		> li {
+			background-image: unset;
 
-		&.active {
-			&.with-menu > a {
-				padding-right: 70px;
+			> a {
+				background-image: unset;
+				background-position: inherit;
 			}
 
-			.calendar-indicator {
-				height: 16px;
+			.sprite::before {
+				content: '';
+				position: absolute;
+				z-index: -1;
 				width: 16px;
-				margin-left: 2px;
-				margin-right: 2px;
-			}
-		}
-
-		&.edit {
-			> a,
-			.app-navigation-entry-utils,
-			.app-navigation-entry-menu {
-				display: none;
+				height: 16px;
+				background: url('../../../apps/tasks/css/../img/sprites.svg') no-repeat;
+				margin: 14px;
+				margin-left: -30px;
+				background-position: inherit;
 			}
 
-			.app-navigation-entry-edit.name {
-				display: inline-block;
-				height: auto;
+			&.active .app-navigation-entry-bullet {
+				height: 16px;
+				width: 16px;
+				margin: 14px;
 			}
-		}
 
-		&.animate-up {
-			max-height: 0;
-			border-bottom: 0;
-			overflow: hidden;
-		}
+			&.edit {
+				> a,
+				.app-navigation-entry-utils,
+				.app-navigation-entry-menu {
+					display: none;
+				}
 
-		&:hover > a span {
-			&.title {
-				opacity: 1;
+				.app-navigation-entry-edit.name {
+					display: inline-block;
+					position: relative;
+				}
 			}
 
-			&.icon {
-				opacity: 1;
+			&.animate-up {
```

Comment:
Doesn't seem to work for hidden elements (see the little dot?):
![image](https://user-images.githubusercontent.com/4129927/33556507-70433708-d904-11e7-8d12-ead146082444.png)
Here's another view:
![screenshot from 2017-12-04 15-06-14](https://user-images.githubusercontent.com/4129927/33556630-d5a65922-d904-11e7-8fb4-61389fc508b4.png)


I'd suggest dropping the animation and just hiding this with `display:none`

Reply:
I will check.

Reply:
I fixed this. `display: none` was the solution.